control/veda_val.py

```diff
@@ -332,13 +304,7 @@ def send_404(self):
             **self.val_data
         )
 
-        r2 = requests.post(
-            self.auth_dict['val_api_url'] + '/',
-            data=json.dumps(sending_data),
-            headers=self.headers,
-            timeout=self.auth_dict['global_timeout']
-        )
-
+        r2 = self.oauth2_client.request('POST', self.auth_dict['val_api_url'] + '/', data=json.dumps(sending_data))
```

Comment:
Nit: It'd be nice to keep a standard way of forming the URL string in all these calls - all using `join`.

Reply:
👍